en/micro-integrator/docs/develop/create-integration-project.md

```diff
@@ -0,0 +1,117 @@
+    <img src="../../assets/img/new-project/create-a-project.png" title="Create a New Integration Project" width="700" alt="Create a New Integration Project"/>
+
+## Sub projects
+
+An integration project can consist of multiple sub-projects. So multiple small projects can exist under a single integration project, where each of these can be dependent on each other and can be grouped together. However, it is not necessary that all sub-projects in an integration project be dependent on every other sub-project.
+
+To add sub-projects to an existing integration project, right click on the integration project and hover over **New** to see the available project creation options.
+
+<img src="../../assets/img/new-project/new-sub-project.png" title="Add a New Sub Project" width="250" alt="Add a New Sub Project"/>
+
+Once you create the new sub project, you can see this nested under your integration project folder in the project explorer.
+
+## About Maven Multi Module integration projects
+
+The Maven Multi Module (MMM) integration project is the parent project in an integration solution and sub-projects can be added under this parent project. By default, an integration project is an MMM project unless you specify otherwise. 
+
+By building the parent MMM project, you can build all the sub-projects in the integration solution simultaneously. This allows you to seamlessly push your integration solutions to a CI/CD pipeline. Therefore, it is recommended as a best practice to create your Config project and other projects inside an MMM integration project.
+
+This allows you to manage multiple projects such as Config projects, Composite Application projects, and Registry Resource projects as a single entity.
+
+Although the recommended approach is to create an integration project, which essentially has all the functionality of an MMM project, you can also create an MMM project separately.
+
+**To create the MMM project**:
+
+1.  Open **WSO2 Integration Studio** and click **New Maven Multi Module Project** in the **Getting Started** view.
+
+2.  In the **Maven Modules Creation Wizard** that opens, enter an artifact ID and other parameters as shown below. The artifact ID will be the name of your MMM project.
+
+    <img src="../../assets/img/create_project/new_maven_multi_module.png" width="500">
+
+3.  Click **Finish**. The MMM project is created in the project explorer. 
+
+    <img src="../../assets/img/create_project/proj_explorer_mmm.png" width="300">
+
+Now you can create other projects inside the MMM project. For example, let's create a **Config** project and a **Composite Application** project.
+
+You can create sub-projects under this parent MMM project.
+
+## Types of projects
```

Comment:
Integration modules?

Reply:
We cannot use "modules" here so it can be "Integration projects" but I think that's confusing.